base/promotion.jl

```diff
@@ -240,7 +251,22 @@ end
 """
     ^(x, y)
 
-Exponentiation operator.
+Exponentiation operator. If `x` is a matrix, computes the matrix exponential.
```

Comment:
when I see "matrix exponential" I tend to think `expm`, maybe it's clear enough from context though